firefox_puppeteer/api/permissions.py

```diff
@@ -0,0 +1,14 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from ..base import BaseLib
+
+
+class Permissions(BaseLib):
+    def remove(self, host, permission):
+        with self.marionette.using_context('chrome'):
```

Comment:
Bug 1088905 has the relevant discussion, cases like these are why we added the context manager. We should add this as we find conflicts. It's going to make more sense for functions like these that do relatively isolated things. For more involved interactions with chrome like the tabs library I don't think we'll find too many callers in content scope, anyway.